.github/workflows/python-app-ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         os: [ macos-latest, ubuntu-latest ]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
Why are we changing this version?

Reply:
Upgrading runtime `node12 --> node16`. It won't break anything, just making sure everything is up-to-date while developing.